streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql

```diff
@@ -101,14 +101,14 @@ create table "public"."t_alert_config" (
   "id" int8 not null default nextval('streampark_t_alert_config_id_seq'::regclass),
   "user_id" int8,
   "alert_name" varchar(128) collate "pg_catalog"."default",
-  "alert_type" int4,
+  "alert_type" int4 default 0,
   "email_params" varchar(255) collate "pg_catalog"."default",
   "sms_params" text collate "pg_catalog"."default",
   "ding_talk_params" text collate "pg_catalog"."default",
   "we_com_params" varchar(255) collate "pg_catalog"."default",
   "http_callback_params" text collate "pg_catalog"."default",
   "lark_params" text collate "pg_catalog"."default",
-  "create_time" timestamp(6) not null,
+  "create_time" timestamp(6) not null default current_timestamp,
   "modify_time" timestamp(6) not null
```

Comment:
Could you confirm that the modify_time field whether need default value?

Reply:
@lvshaokang Thank you for reminding me. `modify_time` can also set the default value, which means that `create_time` and `modify_time` are equal when new records are added, and modify_time must be updated manually for subsequent changes.

Reply:
Thanks for your review, addressed.